language/move-compiler/src/expansion/aliases.rs

```diff
@@ -152,28 +153,51 @@ impl AliasMap {
         self.unused.len()
     }
 
-    pub fn module_alias_get(&mut self, n: &Name) -> Option<&ModuleIdent> {
+    pub fn module_alias_get(&mut self, n: &Name) -> Option<ModuleIdent> {
         match self.modules.get_mut(n) {
             None => None,
             Some((depth_opt, ident)) => {
```

Comment:
nits: Pattern match here:
```
Some((depth_opt, sp!(_, ModuleIdent_ {
              address,
              module: ModuleName(sp!(_, module)),
})
```
or 
```
Some((depth_opt, ident)) => {
   let  sp!(_, ModuleIdent_ { address, module: ModuleName(sp!(_, module))) }) = ident;
```
It is more consistent with what I try to do in the compiler. I find I get lost in long accessor chains